custom/icds/tasks.py

```diff
@@ -159,3 +165,37 @@ def run_user_indicators(phased_rollout=True):
             if is_monday:
                 run_indicator.delay(domain, user_id, LSSubmissionPerformanceIndicator, language_code)
                 run_indicator.delay(domain, user_id, LSVHNDSurveyIndicator, language_code)
+
+
+@periodic_task(run_every=crontab(minute=0, hour='22'))
+def delete_old_images():
+    if settings.SERVER_ENVIRONMENT not in settings.ICDS_ENVS:
+        return
+
+    start = datetime.utcnow()
+    max_age = start - timedelta(days=90)
+    db = get_blob_db()
+    paths = []
+    deleted_attachments = []
+
+    def _get_query(db_name, max_age):
+        return XFormAttachmentSQL.objects.using(db_name).filter(
+            content_type='image/jpeg',
+            form__domain='icds-cas',
+            form__received_on__lt=max_age
+        )
+
+    for db_name in get_db_aliases_for_partitioned_query():
+        attachments = _get_query(db_name, max_age)
+        while attachments.exists():
+            for attachment in attachments[10000]:
```

Comment:
Was `attachments[10000]` meant to be a slice (limit)? Something like `attachments[:10000]` As is, it appears to reference the 10,000th attachment, and I don't understand why you'd only want to that here.